Documentation~/CinemachineBodyOrbitalTransposer.md

```diff
@@ -14,13 +14,13 @@ __Orbital Transposer__ optionally re-centers the camera automatically.  When __R
 
 | **Property:** || **Function:** |
 |:---|:---|:---|
-| __[Binding Mode](CinemachineBindingModes.md)__ || The coordinate space to use when interpreting the offset of the Virtual Camera from the Follow target. |
-| | _Lock To Target On Assign_ | The target’s local frame at the moment when the Virtual Camera was activated, or when the target was assigned. |
-| | _Lock To Target With World Up_ | The target’s local frame, with the tilt and roll reset to zero. |
-| | _Lock To Target No Roll_ | The target’s local frame, with the roll reset to zero. |
-| | _Lock To Target_ | The target’s local frame. |
-| | _World Space_ | World space offset. |
-| | _Simple Follow With World Up_ | Relative to the target, using camera-local axes. |
+| __[Binding Mode](CinemachineBindingModes.md)__ || The coordinate space to use to interpret the offset from the target. |
+| | _Lock To Target On Assign_ | Makes the orientation of the virtual camera match the local frame of the Follow target, at the moment when the virtual camera is activated or when the target is assigned. This offset remains constant in world space. The camera does not rotate along with the target. |
+| | _Lock To Target With World Up_ | Makes the virtual camera use the local frame of the Follow target with tilt and roll set to 0. This binding mode ignores all target rotations except yaw. |
+| | _Lock To Target No Roll_ | Makes the virtual camera use the local frame of the Follow target, with roll set to 0. |
+| | _Lock To Target_ | Makes the virtual camera use the local frame of the Follow target. When the target rotates, the camera rotates with it to maintain the offset and to maintain the same view of the target. |
```

Comment:
A small change (not your fault, it was that way before): I think this is misleading:
"When the target rotates, the camera rotates with it"
because the camera does not rotate.
This would be better:
"When the target rotates, the camera _moves_ with it"

Reply:
Done.